### ISSUE TITLE
Calculation of No. of trainable parameters in ViT

### ISSUE THREAD

**Issue by zhoutianyu16tue** [SHI-Labs/Compact-Transformers#43]
Hi,

I was reading the medium post, https://medium.com/pytorch/training-compact-transformers-from-scratch-in-30-minutes-with-pytorch-ff5c21668ed5.

Once screenshot shows that 
![image](https://user-images.githubusercontent.com/22154134/143434876-57e86459-6d6a-4af2-89a0-60b5766bc5dc.png)
Could you also share the value of **embed_dim**.

With default value of 768, it doesn't seem likely that 0.2M parameters are correct for the model in the first row.

Best,
Tianyu.

**Comment by alihassanijr** [comment]
Hello,
Thank you for your interest.
Embedding dim per head is 64, therefore the smallest model with 2 heads has a total embedding dim of 128.
The largest with 4 heads is 256-dimensional.
I believe these were noted in the paper, and just by loading one of the models, or looking at the source for one of them, it will be clear that none of our models have an embedding dim of 768:

https://github.com/SHI-Labs/Compact-Transformers/blob/e6869b4326e252bf9d6c1e5439ab824f2f4d7f2a/src/vit.py#L83-L100

I'll close this issue for now, but feel free to open it back up in case you have any other questions.
